helpers/webpack.coffee

```diff
@@ -0,0 +1,5 @@
+window.$ = window.jQuery = require "jquery"
+
+$(window)
```

Comment:
I'm not certain if this is correct, not got mobile to run yet to see how the webpack build works. We may not need this, as it should be running in the browser context